string rosmsg_type
string data